packaging/Libuv/project.json

```diff
@@ -0,0 +1,52 @@
+{
+  "version": "1.10.0-*",
+  "description": "Multi-platform support library with a focus on asynchronous I/O.",
+  "licenseUrl": "https://raw.githubusercontent.com/aspnet/libuv-build/775a18ca77368a0f4ca753e82b2369f24707fb3e/build/License.txt",
+  "authors": [
+    "Libuv"
+  ],
+  "packOptions": {
+    "repository": {
+      "type": "git",
+      "url": "http://github.com/aspnet/libuv-build"
+    },
+    "tags": [
+      "libuv"
+    ],
+    "files": {
+      "mappings": {
+        "/": "License.txt",
+        "runtimes/win7-x64/native/": "runtimes/win7-x64/native/*",
+        "runtimes/win7-x86/native/": "runtimes/win7-x86/native/*",
+        "runtimes/win8-arm/native/": "runtimes/win8-arm/native/*",
+        "runtimes/osx/native/": "runtimes/osx/native/*",
+        "runtimes/linux-x64/native/": "runtimes/linux-x64/native/*",
```

Comment:
I don't think so. We added linux/x64 not that long ago but since we basically build once and copy the same binaries for all RIDs we talked about leaving just linux/x64. This is possible because libuv depends only on libc.